pennylane_sphinx_theme/footer.py

```diff
@@ -88,10 +92,6 @@
                     "name": "Glossary",
                     "href": "https://pennylane.ai/qml/glossary/",
                 },
-                {
-                    "name": "FAQ",
-                    "href": "https://pennylane.ai/faq/",
-                },

```

Comment:
What's the reason for removing `FAQ` from the dropdown?
<img width="718" alt="Screenshot 2023-09-25 at 10 34 20 AM" src="https://github.com/PennyLaneAI/pennylane-sphinx-theme/assets/8475900/9d1bf7e2-a9e6-42d5-b72e-f4a26ec0fada">

Reply:
@AndrewGardhouse The FAQ is still in the navbar - it was just removed from the footer!

![image](https://github.com/PennyLaneAI/pennylane-sphinx-theme/assets/5883774/2613586c-415d-4f97-9cba-a871a0736fc3)

Reply:
Ah gotcha 🤦 I should have read the file name... 😂